src/main/java/unicash/ui/MainWindow.java

```diff
@@ -211,7 +211,10 @@ private CommandResult executeCommand(String commandText) throws CommandException
             CommandResult commandResult = logic.execute(commandText);
             logger.info("Result: " + commandResult.getFeedbackToUser());
             resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
-            summaryWindow.setPieChart(logic.getExpenseSummary());
+            summaryWindow.setPieChart(logic.getExpenseSummaryPerCategory());
+            summaryWindow.setLineGraph(logic.getExpenseSummaryPerYearMonth());
+            summaryWindow.setPieChart(logic.getExpenseSummaryPerCategory());
+            summaryWindow.setLineGraph(logic.getExpenseSummaryPerYearMonth());
```

Comment:
Hm, curious but why is this duplicated?

Reply:
It's been removed now! Accidentally left this bit of code in when I was doing some testing earlier. Thanks for pointing this out!